great_expectations/util.py

```diff
@@ -888,6 +888,67 @@ def read_sas(
         )
 
 
+def build_in_memory_runtime_context() -> "BaseDataContext":  # noqa: F821
```

Comment:
I think we can bypass this issue with `if TYPE_CHECKING` and `from future import __annotations__`. Let me know if you need a reference point.

Reply:
@cdkini How about we attempt this in a follow-on effort?

Reply:
I think the lift is very small here - if there is another change to be made (i.e. type hinting with `AbstractDataContext`), I'd ask that we get in this change now